tests/roles/mariadb_copy/tasks/main.yaml

```diff
@@ -12,8 +12,9 @@
       PODIFIED_MARIADB_IP={{ podified_mariadb_ip_result.stdout }}
       MARIADB_IMAGE=quay.io/podified-antelope-centos9/openstack-mariadb:current-podified
 
-      EXTERNAL_MARIADB_IP={{ external_mariadb_ip }}
-      EXTERNAL_DB_ROOT_PASSWORD="{{ external_db_root_password }}"
+      # TODO: remove the default(external_...) when CI is transitioned to use 'source_...'
```

Comment:
👀

Reply:
I should make one more patch to CI itself and then follow up here again, to avoid a chicken-and-egg problem with the var names.